tests/helpers/run_sh_command.py

```diff
@@ -12,8 +13,12 @@ def run_sh_command(command: List[str]):
     """Default method for executing shell commands with pytest and sh package."""
     msg = None
     try:
-        sh.python(command, _out=lambda log: print(log.strip()))
+        with StringIO() as buf:
+            sh.python(command, _out=buf, _env={"NO_COLOR": "1"})
+            output = buf.getvalue()
+        return output
     except sh.ErrorReturnCode as e:
         msg = e.stderr.decode()
-    if msg:
+    # The error message could be empty.
+    if msg is not None:
         pytest.fail(msg=msg)
```

Comment:
This is probably unnecessary for this PR now.

Reply:
I will make a separate PR.

Reply:
https://github.com/IntelLabs/MART/pull/61